tests/testthat/test-endpoints-upload.R

```diff
@@ -47,7 +47,7 @@ test_that("api can upload input files", {
   res2 <- api$request("POST", "/internal/upload/input/survey_data.csv",
                       body = charToRaw(file))
 
-  expect_equal(res, res2)
+  expect_equal(res$body, res2$body)
```

Comment:
These have started failing since we added request_id into the headers. Different value for the header between `res` and `res2`. What I'm not sure about is why this didn't fail in the PR where I added those headers. But hopefully this should resolve it.